contrib/build-container.sh

```diff
@@ -84,6 +85,10 @@ get_options() {
                 missing_requirement $1
             fi
             ;;
+        --ucx-upstream)
+            # Master branch (v1.20) also containing EFA SRD support
+            UCX_REF=3842a65f02a8e2f3635a66189cd82dc86bf065c2
```

Comment:
maybe we will need to change it depending on the ongoing debug

Reply:
yes